.eslintrc.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
   extends: [
     'plugin:vue/recommended',
     '@vue/typescript/recommended',
-    '@vue/prettier',
+    'plugin:storybook/recommended',
+    '@vue/prettier'
```

Comment:
do you still need this if prettier is removed?

Reply:
Yes we do. We are using prettier via ESLINT (using the @vue/prettier config) instead of running prettier as `prettier` itself via the `prettierrc` file.